ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -224,6 +229,12 @@ const BoxHeader = ({text= '', header =  '', subHeader = '', style= {}, ...props}
   </div>;
 };
 
+interface DataSetPreviewList {
```

Comment:
is this used anywhere? I was going to ask about value being a list of lists but I couldn't find usages of this type.

Reply:
This is used in this class. Since its no longer generated (because it is just client side structuring now), I redefined the interface to also include useful information (like isLoading and possibly isErrored)